app/src/main/java/com/github/se/gomeet/ui/navigation/NavigationAction.kt

```diff
@@ -86,6 +92,31 @@ class NavigationActions(val navController: NavHostController) {
     }
   }
 
+  fun navigateToScreen(route: String) {
+    navController.navigate(route)
+  }
+
+  fun navigateToEventInfo(
+      title: String,
+      date: String,
+      time: String,
+      organizer: String,
+      rating: Double,
+      description: String,
+      loc: LatLng
+  ) {
+    val route =
+        Route.EVENT_INFO.replace("{title}", Uri.encode(title))
+            .replace("{date}", Uri.encode(date))
+            .replace("{time}", Uri.encode(time))
+            .replace("{organizer}", Uri.encode(organizer))
+            .replace("{rating}", rating.toString())
+            .replace("{description}", Uri.encode(description))
+            .replace("{latitude}", loc.latitude.toString())
+            .replace("{longitude}", loc.longitude.toString())
+    navController.navigate(route)
+  }
+
```

Comment:
Again, passing an event as a parameter would help a lot. Is it doable?

Reply:
Will be looked into later